### PR TITLE
CNS-309: fix block parsers for cache dr

### PR DESCRIPTION
**Pull request by oren-lava**
* **Please check if the PR fulfills these requirements**
- [ ] Commit message follows the Contribution Guidelines
- [ ] Tests ran locally and added/modified if needed
- [ ] Docs have been added/updated, if applicable
- [ ] If applicable - JIRA ticket ID was added

* **What kind of change does this PR introduce?** (Bug fix, feature, unit tests, docs update, ...)
ETH spec update

* **What is the current behavior?** (You can also link to an open issue here)
ETH spec didn't have correct parsing functions

* **What is the new behavior (if this is a feature change)?**
fix the issue

* **Please describe what manual tests you ran, if applicable**

* **Other information**:

### CODE REVIEW

File: cookbook/spec_add_ethereum.json [hunk 1]
```diff
@@ -170,16 +170,16 @@
...
-                            "parser_func": "EMPTY"
+                            "parser_func": "DEFAULT"
                         },
                         "compute_units": "1",
                         "enabled": true,
                         "api_interfaces": [
                             {
                                 "category": {
-                                    "deterministic": false,
+                                    "deterministic": true,
```
Comment (Yaroms) [inline cookbook/spec_add_ethereum.json:182]: this is not deterministic
https://docs.infura.io/infura/networks/ethereum/json-rpc-methods/eth_hashrate
can change per node

File: cookbook/spec_add_ethereum.json [hunk 2]
```diff
@@ -194,16 +194,16 @@
...
```
Comment (Yaroms) [inline cookbook/spec_add_ethereum.json:206]: also not deterministic

**Review by Yaroms — CHANGES_REQUESTED** [review]

File: cookbook/spec_add_ethereum.json (hunk 2 above)
Comment (oren-lava) [reply cookbook/spec_add_ethereum.json:206]: done

File: cookbook/spec_add_ethereum.json (hunk 1 above)
Comment (oren-lava) [reply cookbook/spec_add_ethereum.json]: done

**Review by Yaroms — APPROVED** [review]

**Pull request merged**